servers/src/main/java/tachyon/master/journal/JournalTailerThread.java

```diff
@@ -79,8 +79,17 @@ public void run() {
         long waitForShutdownStart = -1;
 
         // Load the checkpoint file.
+        LOG.info("Waiting to load the checkpoint file.");
         JournalTailer tailer = new JournalTailer(mMaster, mJournal);
+        while (!tailer.checkpointExists()) {
```

Comment:
Ok. Should there be some sort of sleep here, to avoid overloading the ufs operations?